packages/ketcher-react/src/script/ui/views/components/ContextMenu/ContextMenuTrigger.tsx

```diff
@@ -81,7 +81,8 @@ const ContextMenuTrigger: React.FC<PropsWithChildren> = ({ children }) => {
       let triggerType: ContextMenuTriggerType
 
       if (!closestItem) {
-        if (selection) {
+        const isLeftMouseButtonPressed = event.buttons === 1
+        if (selection && !isLeftMouseButtonPressed) {
```

Comment:
It takes a while to think why `isLeftMouseButtonPressed` is needed. Maybe we can:
```suggestion
        const isLeftMouseButtonPressed = event.buttons === 1
        const isRotationReverted = isLeftMouseButtonPressed
        if (selection && !isRotationReverted) {
```